dfx.json

```diff
@@ -64,6 +64,25 @@
 				}
 			}
 		},
+		"cycles_ledger": {
+			"type": "custom",
+			"candid": "https://github.com/dfinity/cycles-ledger/releases/download/cycles-ledger-v1.0.1/cycles-ledger.did",
+			"wasm": "https://github.com/dfinity/cycles-ledger/releases/download/cycles-ledger-v1.0.1/cycles-ledger.wasm.gz",
+			"init_arg": "( variant { Init = record { index_id = null; max_blocks_per_request = 9_999 : nat64 }},)",
+			"remote": {
+				"id": {
+					"ic": "um5iw-rqaaa-aaaaq-qaaba-cai"
+				}
+			}
+		},
+		"cycles_depositor": {
+			"dependencies": ["cycles_ledger"],
+			"type": "custom",
+			"build": "scripts/build.cycles_depositor.sh",
```

Comment:
We can use `candid` and `wasm` entries because the arguments need to be dynamic?